spa/src/pages/ConfigSteps/index.tsx

```diff
@@ -137,76 +132,21 @@ const ConfigSteps = () => {
 		});
 	};
 
-	const getOrganizations = useCallback(async () => {
+	const getOrganizations = async (autoRedirectToInstall = true) => {
```

Comment:
Switched this back to just normal method instead of using callback and using `// eslint-disable`, otherwise there are just too many eslint errors which is happening because of the multiple usage of methods within each other.